bigbluebutton-html5/private/locales/pt_BR.json

```diff
@@ -45,6 +45,7 @@
     "app.media.screenshare.safariNotSupported": "Atualmente, o compartilhamento de tela não é suportado pelo Safari. Por favor, use o Firefox ou o Google Chrome.",
     "app.meeting.ended": "Esta reunião terminou",
     "app.meeting.endedMessage": "Você será redirecionado para a tela inicial",
+    "app.presentation.close": "Fechar apresentação",
```

Comment:
It's best to only add the en.json new lines and let the translations go through Transifex so there are no conflicts when pulling the locales from there

Reply:
Should I just remove these ones now?